src/TileSourceWebMapService.cpp

```diff
@@ -310,8 +310,8 @@ std::string TileSourceWebMapService::loadData(int level, int x, int y) {
 
   url.precision(std::numeric_limits<double>::max_digits10);
   url << mUrl << "&version=1.1.0&request=GetMap&tiled=true&layers=" << mLayers
-      << "&styles=" << mStyles << "&bbox=" << x * size << "," << y * size << "," << x * size + size
-      << "," << y * size + size << "&width=257&height=257&srs=EPSG:900914&format=" << format;
+      << "&bbox=" << x * size << "," << y * size << "," << x * size + size << "," << y * size + size
+      << "&width=257&height=257&srs=EPSG:900914&format=" << format;
```

Comment:
Would this look nicer with `fmt`?

Reply:
It would, but I think it's not in the scope of this PR. This is a quite important part and would need to do some in-depth testing for precision issues.